tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+minversion = 3.18.0
+skipsdist = True
+envlist = bashate
+
+[testenv]
+usedevelop = False
+basepython = python3
+allowlist_externals = bash
+commands = bash {toxinidir}/setup_dependencies.sh
+
+[testenv:bashate]
+deps =
+   {env:BASHATE_INSTALL_PATH:bashate==2.0.0}
+allowlist_externals = bash
+commands = bash -c "find {toxinidir}             \
+         -not \( -type d -name .?\* -prune \)    \
+         -type f                                 \
+         -not -name \*~                          \
+         -not -name \*.md                        \
+         \(                                      \
+          -name \*.sh                            \
+         \)                                      \
+         -print0 | xargs -0 bashate -v -iE006 -eE005,E042"
+
+[testenv:run_unlimited_bw_demo]
```

Comment:
shouldn't we have a target for the other bandwidth scenario ?

Reply:
I'll hold onto that for when we're done with all the examples. I'll tackle whatever more needed ones in a follow-up PR.